### PR TITLE
fixes #271

### LINKED ISSUE #271: Importing fails when IMPORT_WIKIPEDIA=true
Using CIVO k3s cluster for testing.

2021-11-27 16:11:29: Importing wikipedia importance data
--
2021-11-27 18:11:30 | Traceback (most recent call last):
2021-11-27 18:11:30 | File "/usr/local/bin/nominatim", line 11, in <module>
2021-11-27 18:11:30 | exit(cli.nominatim(module_dir='/usr/local/lib/nominatim/module',
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/cli.py", line 288, in nominatim
2021-11-27 18:11:30 | return parser.run(**kwargs)
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/cli.py", line 92, in run
2021-11-27 18:11:30 | return args.command.run(args)
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/clicmd/setup.py", line 103, in run
2021-11-27 18:11:30 | if refresh.import_wikipedia_articles(args.config.get_libpq_dsn(),
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/tools/refresh.py", line 139, in import_wikipedia_articles
2021-11-27 18:11:30 | execute_file(dsn, datafile, ignore_errors=ignore_errors,
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/db/utils.py", line 47, in execute_file
2021-11-27 18:11:30 | remain = _pipe_to_proc(proc, fdesc)
2021-11-27 18:11:30 | File "/usr/local/lib/nominatim/lib-python/nominatim/db/utils.py", line 14, in _pipe_to_proc
2021-11-27 18:11:30 | chunk = fdesc.read(2048)
2021-11-27 18:11:30 | File "/usr/lib/python3.8/gzip.py", line 292, in read
2021-11-27 18:11:30 | return self._buffer.read(size)
2021-11-27 18:11:30 | File "/usr/lib/python3.8/_compression.py", line 68, in readinto
2021-11-27 18:11:30 | data = self.read(len(byte_view))
2021-11-27 18:11:30 | File "/usr/lib/python3.8/gzip.py", line 479, in read
2021-11-27 18:11:30 | if not self._read_gzip_header():
2021-11-27 18:11:30 | File "/usr/lib/python3.8/gzip.py", line 427, in _read_gzip_header
2021-11-27 18:11:30 | raise BadGzipFile('Not a gzipped file (%r)' % magic)
2021-11-27 18:11:30 | gzip.BadGzipFile: Not a gzipped file (b'<h')

### PR DESCRIPTION
**Pull request by norcis**

### CODE REVIEW

File: 3.7/init.sh [hunk 1]
```diff
@@ -4,27 +4,27 @@ OSMFILE=${PROJECT_DIR}/data.osm.pbf
...
+  curl https://www.nominatim.org/data/us_postcode_data.sql.gz -L -o ${PROJECT_DIR}/us_postcode_data.sql.gz
 else
   echo "Skipping optional US postcode import"
 fi;
 
 
 if [ "$PBF_URL" != "" ]; then
 	echo Downloading OSM extract from "$PBF_URL"
-	curl -L "$PBF_URL" --create-dirs -o $OSMFILE
+	curl -L "$PBF_URL" -L --create-dirs -o $OSMFILE
```
Comment (leonardehrenfried) [inline 3.7/init.sh:27]: This one already has the option at the beginning. Not sure if it will cause an error when set twice.

Comment (norcis) [reply 3.7/init.sh]: thanks for noticing

**Comment by leonardehrenfried** [comment]
There seems to be an unrelated CI failure in the 3.6 folder. I will check that out tomorrow but the PR looks solid.

Thanks for the contribution.

**Comment by leonardehrenfried** [comment]
I think this might be fixing the problem: https://github.com/mediagis/nominatim-docker/pull/273/commits/e6b16cc593c9b2df3b417fb262944d23d4fcf94c

**Comment by leonardehrenfried** [comment]
Please rebase onto master and the 3.6 build should work.

**Comment by leonardehrenfried** [comment]
Scratch that. I merged it anyway and fill fix problems on master, if there are any.

Thanks.

**Comment by leonardehrenfried** [comment]
The merge went ok and your changes work, however there is a problem in the build pipeline that is preventing us from uploading docker images.

On ARM the Ubuntu image segfaults when downloading the code from nominatim.org. @philipkozeny and I are trying to figure out how to proceed.

**Comment by leonardehrenfried** [comment]
We've disabled the ARM build and your change is now on Dockerhub.

**Pull request merged**